packages/modeling/src/operations/extrusions/extrudeHelical.test.js

```diff
@@ -0,0 +1,60 @@
+import test from 'ava'
+import { TAU } from '../../maths/constants.js'
+import { geom2, geom3 } from '../../geometries/index.js'
+import { circle } from '../../primitives/circle.js'
+
+import { extrudeHelical } from './extrudeHelical.js'
```

Comment:
For tests... imports should be from index, as that verifies that the index is functionally correct.